nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/summary/nf-summary.js

```diff
@@ -180,6 +180,16 @@ nf.Summary = (function () {
                                 'width': $(window).width() + 'px'
                             });
 
+                            var summaryTableHeight = $('#summary').outerHeight() -
+                                ($('#summary-header-text').outerHeight() + 30) - //30 for margin-bottom on summary-header-text
+                                ($('#summary-tabs').outerHeight() + 20) - //20 for margin-bottom on tab-container
+                                $('#summary-filter-controls').outerHeight() -
+                                $('#flow-summary-refresh-container').outerHeight();
+                            if (top === window) {
+                                summaryTableHeight -= 80; //80 for padding
+                            }
+                            $('.summary-table').height(summaryTableHeight).show();
```

Comment:
I think this logic is unnecessary as the provenance table is bound from the bottom using position absolute. The height should adjust due to the CSS rules.